$MirOS: ports/print/cups/patches/patch-cgi-bin_admin_c,v 1.1.7.1 2005/03/18 15:51:00 tg Exp $
--- cgi-bin/admin.c.orig	Tue Apr 18 19:58:42 2006
+++ cgi-bin/admin.c	Tue May  9 17:39:25 2006
@@ -718,7 +718,7 @@ do_am_printer(http_t *http,		/* I - HTTP
         break;
       else
       {
-        sprintf(baudrate, "%d", baudrates[i]);
+        snprintf(baudrate, sizeof(baudrate),"%d", baudrates[i]);
 	cgiSetArray("BAUDRATES", i, baudrate);
       }
 
